### PR TITLE
ARCHCNL-10: Improve test coverage in kotlin parser

### PR DESCRIPTION
**Pull request by AlexHamburg**

### CODE REVIEW

File: kotlin-parser/src/test/java/org/archcnl/kotlinparser/parser/ModelExtractorTest.java [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+  private final String getType = "GET";
+  private final String complexClass = "ComplexClass";
+  private final String classWithInnerClass = "ClassWithInnerClass";
+  private final String innerClass = "InnerClass";
+  private final String classInSubpackage = "ClassInSubpackage";
+  private final String anotherClass = "AnotherClass";
+
+  private final String baseNamespace = "example";
+
+  private ModelExtractor classUnderTest;
```
Comment (toklinke) [inline kotlin-parser/src/test/java/org/archcnl/kotlinparser/parser/ModelExtractorTest.java:30]: The variable `classUnderTest` does not need to be in class scope but should be moved to the individual test methods.

Comment (toklinke) [reply kotlin-parser/src/test/java/org/archcnl/kotlinparser/parser/ModelExtractorTest.java:30]: Moreover, the name is not very meaningful: Why not just call it `modelExtractor`?

Comment (AlexHamburg) [reply kotlin-parser/src/test/java/org/archcnl/kotlinparser/parser/ModelExtractorTest.java]: @toklinke it's about "matter of taste" in my opinion. You can see based on the type of a variable, what class is under test. But I adjusted it)

Comment (toklinke) [reply kotlin-parser/src/test/java/org/archcnl/kotlinparser/parser/ModelExtractorTest.java]: :+1:

**Review by toklinke — APPROVED** [review]

**Pull request merged**